tests/toolkit/matl_sell_policy_tests.cc

```diff
@@ -100,9 +100,23 @@ TEST_F(MatlSellPolicyTests, Bids) {
   ASSERT_EQ(obs.size(), 1);
   ASSERT_EQ((*obs.begin())->bids().size(), 2);
   ASSERT_FLOAT_EQ((*(*obs.begin())->bids().begin())->offer()->quantity(),
-                  mat->quantity() / 2);
+                  qty / 2);
+  ASSERT_EQ((*(*obs.begin())->bids().begin())->offer()->comp(), comp1);
+
+  // If available quantity is bigger than the quanta only an integer number of quanta is offered
+  p.Init(NULL, &buff, "", qty, true, 2).Set(commod);
+  obs = p.GetMatlBids(reqs);
+  ASSERT_EQ(obs.size(), 1);
+  ASSERT_EQ((*obs.begin())->bids().size(), 1);
+  ASSERT_FLOAT_EQ((*(*obs.begin())->bids().begin())->offer()->quantity(),
+                  2);
```

Comment:
I'm not sure I understand why this test behaves differently from the previous one???  The initializer for `p` is the same except for the value of the final entry - why only a single bid in this case?

Reply:
there is a single bid because the available quantity is 3, the throughput is 3 but the quanta is 2, so only 1 single shipment can fulfill the quanta restriction